profiling_dashboard/templates/profiling_dashboard/base.html

```diff
@@ -1,7 +1,9 @@
+{% load staticfiles %}
+
 <!DOCTYPE html>
 <html>
 <head>
-    <link rel="stylesheet" href="http://twitter.github.com/bootstrap/1.4.0/bootstrap.min.css">
+    <link rel="stylesheet" href="{% static 'css/bootstrap.min.css' %}">
```

Comment:
I think the css file should go in `/profiling_dashboard/css/bootstrap.min.css` - this way it won't conflict with user static files.